src/common/UserObjectType.js

```diff
@@ -0,0 +1,6 @@
+export type UserObjectType = {
+  username: ?string,
+  email: ?string,
+  id: ?string,
+  profile: ?Object,
```

Comment:
This type should describe the returned object from any method that should give you a user object. Meaning IMO that the profile field could be defined by the using application however it would like. This is true for meteor as well. Generics will allow the user to have auto completion for his own defined profile object.. Not a must, but could be cool to have.